src/pages/index.js

```diff
@@ -59,16 +90,67 @@ function openImageModel(event) {
   imagePopup.open(event);
 }
 
-cardSection.renderItems(initialCards);
+function openEditProfilePicture() {
+  profilePicturePopup.open();
+}
+
+function openConfirmDelete(locationCardId, locationCard) {
+  const deletePopup = new PopupDelete(
```

Comment:
Every popup should be created only one time and `setEventListeners` should be called also once, otherwise you add listeners again and again to the same elements. This may cause a **`memory leak`**